web/views.py

```diff
@@ -24,7 +24,8 @@ def add_skill_time(request, pk):
 
 @require_http_methods(['POST'])
 def create_skill(request):
-  user = User.objects.first()
+  #Need User object of requestor
```

Comment:
I prefer to have a space between the "#" and the rest of the comment. (i.e. `# Need requestor's User object`.